.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
```

Comment:
always good to add pypy (and pypy3) to the list as it's a completely different interpreter, it can raise some interesting issues that you wouldn't see normally